setup_cfg_fmt.py

```diff
@@ -198,15 +198,21 @@ def _python_requires(
 
 def _requires(cfg: configparser.ConfigParser, which: str) -> List[str]:
     raw = cfg.get('options', which, fallback='')
+    stripped = raw.strip()
+    if not stripped:
+        return []
 
-    install_requires = raw.strip().splitlines()
+    if raw.startswith(os.linesep):
+        install_requires = stripped.splitlines()
+    else:
+        install_requires = [r.strip() for r in stripped.split(';')]
 
     normalized = sorted(
         (_normalize_req(req) for req in install_requires),
         key=lambda req: (';' in req, _req_base(req), req),
     )
 
-    if len(normalized) > 1:
+    if len(normalized) > 1 or ';' in normalized[0]:
```

Comment:
Maybe it sounds just always do this :thinking:

Reply:
sure, updated